pkg/install/v1alpha1/openebs_crds.go

```diff
@@ -134,6 +134,30 @@ spec:
     # shortNames allow shorter string to match your resource on the CLI
     shortNames:
     - csp
+  additionalPrinterColumns:
+  - JSONPath: .status.capacity.used
+    name: Allocated
+    description: The amount of storage space within the pool that has been physically allocated
+    type: string
+  - JSONPath: .status.capacity.free
+    name: Free
+    description: The amount of free space available in the pool
+    type: string
+  - JSONPath: .status.capacity.total
+    name: Capacity
+    description: Total size of the storage pool
+    type: string
+  - JSONPath: .status.phase
+    name: Status
+    description: Identifies the current health of the pool
+    type: string
+  - JSONPath: .spec.poolSpec.poolType
+    name: Type
+    description: The type of the storage pool
+    type: string
+  - JSONPath: .metadata.creationTimestamp
```

Comment:
This might print some value like -- `2018-12-17T20:16:40Z` ( Not sure what is the name of this format)
Does it print the age?

Reply:
Type `date` will take care of